src/store/sagas/newsSaga.js

```diff
@@ -0,0 +1,20 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+
+import * as ACTION_TYPES from '../../constants/actionTypes';
+import { getNewsSuccess, getNewsFailed } from '../action/newsActions';
+import getNews from '../../api/newsApi';
```

Comment:
`import getNews from '../../api/newsApi';` - suppose that this file would have more than one method